index.html

```diff
@@ -136,6 +136,164 @@ <h2>
+        triggers, joysticks, thumbsticks, touch surfaces) and output controls
+        (haptic actuators). A [=gamepad=] is <dfn>available</dfn> if the [=user
+        agent=] can read the current state of its input controls. A [=gamepad=]
+        that is not [=available=] is <dfn>unavailable</dfn>. The collection of
+        input and output controls for a [=gamepad=] cannot change while the
+        [=gamepad=] is [=available=]. Each input control has one or more
+        <dfn>input values</dfn> that may update over time.
+      </p>
+      <p>
+        The [=user agent=] is responsible for enumerating [=available=]
+        [=gamepads=], detecting when [=gamepads=] become [=available=] or
+        [=unavailable=], and detecting when input controls have updated input
+        values.
+      </p>
+      <p>
+        A [=gamepad=] has a <dfn>gamepad identifier string</dfn>, a
+        human-readable string that identifies the brand or style of
+        [=gamepad=]. The content is decided by the [=user agent=].
+      </p>
+      <p>
+        A [=gamepad=] may be a physical device. If so, it has an <dfn>input
+        control layout</dfn> that describes the position, orientation and type
+        of each input control on the physical device. The [=user agent=] is
+        responsible for recognizing when an [=input control layout=]
+        <dfn data-lt="corresponds with">corresponds with a standard
+        layout</dfn>, meaning the relative positions and orientations of its
+        input controls are close enough to the standard layout that a user can
+        use it interchangeably with other devices that have the same input
+        controls and correspond with the same layout.
+      </p>
+      <p>
+        A [=gamepad=] may have [=axis=] inputs. An <dfn>axis</dfn> is an input
+        control with the following [=input values=]:
+      </p>
+      <ul>
+        <li>A <dfn>logical axis value</dfn>, an integer value that represents
+        the current displacement of the control from a reference position.
+        </li>
+        <li>A <dfn>minimum axis value</dfn>, the smallest in-bounds [=logical
+        axis value=].
+        </li>
+        <li>A <dfn>maximum axis value</dfn>, the largest in-bounds [=logical
+        axis value=].
+        </li>
+      </ul>
+      <p>
+        A [=gamepad=] has an <dfn>axis list</dfn> which is a [=list=]
+        containing all the [=axis=] inputs for the [=gamepad=] in some order
+        determined by the [=user agent=].
+      </p>
+      <p>
+        An input control may be designed to automatically return an [=axis=] to
+        a center position when the user stops interacting with the input
+        control. If so, the [=axis=] has a <dfn>preferred axis state</dfn>. An
+        An [=axis=] with a [=preferred axis state=] may also have a <dfn>center
+        position value</dfn> [=input value=] which is the [=logical axis
+        value=] when the [=axis=] is centered.
```

Comment:
Remove double "An" and maybe we remove the `[=input value=]` reference to make the text more fluid? Or maybe we can have `<dfn>center position input value</dfn>`?
```suggestion
        control. If so, the [=axis=] has a <dfn>preferred axis state</dfn>. An
        [=axis=] with a [=preferred axis state=] may also have a <dfn>center
        position value</dfn> which is the [=logical axis
        value=] when the [=axis=] is centered.
```

Reply:
This same feedback also applies to `<dfn>digital button value</dfn>` and `dfn>button touch value</dfn>`.

Reply:
It needs to be clear which values are [=input values=] and which are just properties of the input control. I prefer tagging them with the cross-reference as it leaves no room for ambiguity. I'll try rewording to make this flow better while keeping [=input value=].